### ISSUE TITLE
Did the fix work?

### ISSUE THREAD

**Issue by IstoraMandiri** [hculap/app-dump#1]
Hi, would appreciate if you opened an issue in the main repo you found a bug. I will happily accept the PR.

Cheers!

**Comment by hculap** [comment]
Yes fix works. I've made new issue in your repo. Sorry that i copied your code and made new package but i was in need. Now you can fix it and I'll remove my version of this package ;) So thanks for you work. It is very good package and I'm sure a lot of meteor developers will use it for their projects.

**Comment by IstoraMandiri** [comment]
No problem @hculap -- just checking!

Many thanks for your help.

Btw, do you think it would be possible for you to put a link in your readme to the original so people don't get confused?

Cheers.
